iphone/cli/commands/_build.js

```diff
@@ -3177,7 +3177,7 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 			outputPaths: [],
 			runOnlyForDeploymentPostprocessing: 0,
 			shellPath: '/bin/sh',
-			shellScript: '"cp -rf \\"$PROJECT_DIR/ArchiveStaging\\"/ \\"$TARGET_BUILD_DIR/$PRODUCT_NAME.app/\\""',
+			shellScript: '"/bin/cp -rf \\"$PROJECT_DIR/ArchiveStaging\\"/ \\"$TARGET_BUILD_DIR/$PRODUCT_NAME.app/\\""',
```

Comment:
There is a performance issue with standalone commands. Apple even recommends to [not use them](https://developer.apple.com/library/content/documentation/OpenSource/Conceptual/ShellScripting/performance/performance.html#//apple_ref/doc/uid/TP40004268-TP40003520-SW14) and gives some intersting benchmarking-tests. I don't this issue is present for many users, but @janvennemann or @sgtcoolguy may have some more insight here.

Reply:
I was unaware of a performance difference. Being that it's called a "shellScript", I assumed Xcode was just spawning the command in a shell in which case I don't see why `/bin/cp` would be significantly slower than `cp`. In the Apple docs, they use the example of the `echo` command which writes to stdout and perhaps that's where the speed issue is. `cp` doesn't output anything unless the verbose flag is set. So, I'm not terribly concerned with performance.

I guess I'm more concerned why `/bin` isn't in your `PATH`.

Reply:
Agreed, we are talking about one iteration of the command here, so i think we can neglect the performance impact, which seems to be a few ms at most according to @rlustemberg benches.

Reply:
Then let‘s take it for 7.0.1!